README.md

```diff
@@ -1,4 +1,61 @@
 
 ## Overview
 
-The `enhancements` repo is made available to discuss specific new features or architectural improvements for the community around `open-cluster-management`.
\ No newline at end of file
+The `enhancements` repo is made available to discuss specific new features or architectural improvements for the community around `open-cluster-management`.
+
+Inspired by the [Kubernetes enhancement]{https://github.com/kubernetes/enhancements} process.
+
+This repository provides a rally point to discuss, debate, and reach consensus for how Open Cluster Management enhancements are introduced.
+
+Enhancements may take multiple releases to ultimately complete and thus provide the basis of a community roadmap. Enhancements may be filed from anyone in the community, but require consensus from domain specific project maintainers in order to implement and accept into the release.
+
+For template references, see the [template]{https://github.com/open-cluster-management/enhancements/blob/master/guidelines/enhancement_template.md}.
+
+## Is My Thing an Enhancement?
+
+A rough heuristic for an enhancement is anything that:
+
+- needs significant effort or changes in a significant way (ex. something that would take 10 person-weeks to implement, introduce or redesign a system component, or introduces API changes)
```

Comment:
semi-related.  is there a deprecation policy on APIs?  If not, I think that's worth writing down.

Reply:
we can reference kube api deprecation policy as a reference for now here https://github.com/open-cluster-management/enhancements/pull/3/files#diff-440cabdb4e100367f3b10442db96ad1c3d688ac9ed5273aee8d89f00b9616147R137